tests/locustfile.py

```diff
@@ -0,0 +1,36 @@
+from locust import HttpUser, task
+from random import shuffle
+
+models = [
+    'user',
+    'media-file',
+    'collection',
+    'clams-app',
+    'pipeline',
+    'batch',
+    'clams-event',
+]
+
+
+class HomePageUser(HttpUser):
+    @task
+    def get_home(self):
+        self.client.get('/')
+
+
+class AdminPageUser(HttpUser):
+    @task
+    def get_admin(self):
+        self.client.get('/admin/')
+
+    @task
+    def get_admin_with_redirect(self):
+        self.client.get('/admin')
+
+    @task
+    def list_models(self):
+        model_list = models.copy()
+        shuffle(model_list)
+        with self.client.rename_request('admin/[app]/list'):
```

Comment:
is this `with` required?

Reply:
It came from the [examples in the docs](https://docs.locust.io/en/stable/writing-a-locustfile.html).

Here, using `with ... rename_request` puts all theses requests under a single header `'admin/[app]/list'` in the report.

Not necessary, but will be helpful grouping tests together once we have many different types.